### DISCUSSION TITLE
[beginner question] Issue with loading file

### DISCUSSION THREAD

**Discussion by mslijepc** [WerWolv/ImHex#1535] [Get Help] [answered]
Hi, 

I just started using Imhex, and I have an issue with loading files. If I load a file in which I assume that data is in `hex` format, e.g. `00 04 2A` it assumes it is `ASCII` and it interprets those characters as (hex) `30 30 20 30 34 20 ...`
Could you please tell me what to do, how to interpret this file as `0x00 0x04 0x2A` ?

Also, I guess some kind of `Getting started guide`, with few basic examples would be nice to have.. (E.g. this post was pretty useful for me https://github.com/WerWolv/ImHex/discussions/365)

**Comment by WerWolv** [comment, marked as answer]
Hey

ImHex is a hex editor which means opening files in it, generally, shows you the raw byte content of that file. It doesn't interpret the data you're opening in any way. For your case, you currently would need to transform the ASCII Hex representation of that data into raw bytes first before you can open it.

About the getting started guide, please check out the documentation which is linked on the welcome screen and make sure you're using the latest update. There's both an interactive introduction tutorial as well as Achievements that are meant to guide you through the various features of ImHex.
